src/main/java/org/numenta/nupic/model/Cell.java

```diff
@@ -142,6 +143,18 @@ public String toString() {
     }
 
     /**
+     * Converts a Collection of {@link Cell}s to {@link Column} indexes.
+     *
+     * @param cells             the list of cells to convert
+     *
+     * @return  sorted array of column indices.
+     */
+    public static int[] asSparseSDR(Collection<Cell> cells) {
+        return cells.stream().mapToInt(c -> c.getColumn().getIndex())
+                .sorted().distinct().toArray();
+    }
+
```

Comment:
Hey @fergalbyrne Thanks for all of your excellent work!

This one however is a nit. Cell has no idea what an SDR is because it is a subordinate concept just like asking a Leaf class to produce artifacts related to Branches? I would move this method to SDR.java perhaps?